pkg/restapi/issuer/operation/operations_test.go

```diff
@@ -151,6 +151,40 @@ const assuranceData = `{
 	  }
 	}`
 
+const validVP = `{
+		"@context": [
+			"https://www.w3.org/2018/credentials/v1",
+			"https://www.w3.org/2018/credentials/examples/v1"
+		],
+		"id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
+		"type": "VerifiablePresentation",
+		"verifiableCredential": [{
+			"@context": [
+				"https://www.w3.org/2018/credentials/v1",
+				"https://www.w3.org/2018/credentials/examples/v1"
+			],
+			"id": "http://example.edu/credentials/1872",
+			"type": "VerifiableCredential",
+			"credentialSubject": {
+				"id": "did:example:ebfeb1f712ebc6f1c276e12ec21"
+			},
+			"issuer": {
+				"id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
+				"name": "Example University"
+			},
+			"issuanceDate": "2010-01-01T19:23:24Z",
+			"credentialStatus": {
+				"id": "https://example.gov/status/24",
+				"type": "CredentialStatusList2017"
+			}
+		}],
+		"holder": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"refreshService": {
+			"id": "https://example.edu/refresh/3732",
+			"type": "ManualRefreshService2018"
```

Comment:
What's `ManualRefreshService2018` and `CredentialStatusList2017`?

Reply:
I remove them